mythical-creatures/exercises/direwolf.js

```diff
@@ -0,0 +1,46 @@
+function createMyDirewolf(name, home, size) {
+  return {
+    name: name,
+    home: home || 'Beyond the Wall',
+    size: size || 'Massive',
+    starksToProtect: [],
+    protect: function(stark) {
```

Comment:
I worry that having methods in objects like this is too much OOP. I'm curious what others think! If no one else reviews this, maybe post in the #frontend-curriculum-updates channel with a code snippet and ask what they think.